src/server/zset_family.cc

```diff
@@ -2370,6 +2432,71 @@ void ZSetFamily::ZPopMinMax(CmdArgList args, bool reverse, ConnectionContext* cn
   OutputScoredArrayResult(result, range_params, cntx);
 }
 
+void ZSetFamily::GeoAdd(CmdArgList args, ConnectionContext* cntx) {
+  string_view key = ArgS(args, 0);
+
+  // TODO: to handle options and multiple elements
+  ZParams zparams;
+
+  string_view longitude = ArgS(args, 1);
+  string_view latitude = ArgS(args, 2);
+  string_view member = ArgS(args, 3);
+
+  // TODO: to remove this check once the TODO above is handled.
+  if (args.size() != 4) {
+    return (*cntx)->SendError(kSyntaxErr);
+  }
+
+  pair<double, double> longlat;
+  for (int i = 0; i < 1; i++) {
```

Comment:
What's the purpose of this `for` loop?
I guess it's somewhat like a placeholder for expanding the functionality per the above `TODO`s. I'd refrain from doing so, and just code the code as it currently stands. Same below with `members` and the single `emplace()` call.
Your call if you agree

Reply:
I am not going to work on geo code anymore. I add these placeholders to create starter tasks. I believe it gives a person some hint on how to extend.

Reply:
Ok, sure. As such, consider adding TODOs as now it's confusing, at least to me :)